src/tmx/TmxUtils/test/J2735MessageTest.cpp

```diff
@@ -542,6 +542,106 @@ TEST_F(J2735MessageTest, EncodeBasicSafetyMessage)
 }
 
 
+
+TEST_F(J2735MessageTest, EncodeBasicSafetyMessag_PartII)
+{	
+	BasicSafetyMessage_t* message = (BasicSafetyMessage_t*) calloc(1, sizeof(BasicSafetyMessage_t) );
```

Comment:
Please adjust the indents of the lines in this function so that some lines don't have an additional indent.

Reply:
Resolved since the indentations only seem to show up on GitHub, not in vscode.